unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRendererHelpers/AvatarRendererHelpers.cs

```diff
@@ -4,6 +4,7 @@ namespace DCL
 {
     public static class AvatarRendererHelpers
```

Comment:
```suggestion
    public static class AvatarRendererUtils
```
As discussed the static class with pure methods should have the `Utils` suffix.